tools/cockpit.spec

```diff
@@ -580,6 +582,9 @@ getent passwd cockpit-ws >/dev/null || useradd -r -g cockpit-ws -d / -s /sbin/no
 # firewalld only partially picks up changes to its services files without this
 test -f %{_bindir}/firewall-cmd && firewall-cmd --reload --quiet || true
 
+# link the inactive issue if it doesn't already exist
+%{_bindir}/ln -s %{_sysconfdir}/%{name}/inactive.issue %{_sysconfdir}/%{name}/cockpit.issue || true
```

Comment:
This also needs a corresponding command in the post-removal script to clean up the symlink on package removal. On Debian, this would be tools/debian/cockpit-ws.postrm with something like
```sh
#!/bin/sh -e
if [ "$1" = purge ] && [ -L /etc/cockpit/cockpit.issue ]; then
    rm -f /etc/cockpit/cockpit.issue
fi
```